src/tests/test_dte_data_models.py

```diff
@@ -782,33 +846,41 @@ def test_init_fail_tipo_documento_ref_invalid(self) -> None:
 
         obj = self.obj_1
 
-        with self.assertRaises(pydantic.v1.ValidationError) as assert_raises_cm:
+        with self.assertRaises(pydantic.ValidationError) as assert_raises_cm:
             dataclasses.replace(
                 obj,
                 tipo_documento_ref="8001",
             )
         self.assertEqual(
-            assert_raises_cm.exception.errors(),
+            assert_raises_cm.exception.errors(
+                include_context=False,
+                include_input=False,
+                include_url=False,
+            ),
             [
                 {
                     'loc': ('tipo_documento_ref',),
-                    'msg': '("The length of \'tipo_documento_ref\' must be a '
-                    'value between 1 and 3", \'8001\')',
+                    'msg': 'Value error, ("The length of \'tipo_documento_ref\' must be a value '
+                    'between 1 and 3", \'8001\')',
```

Comment:
Reformat to reduce diff size.

```suggestion
                    'msg': 'Value error, ("The length of \'tipo_documento_ref\' must be a '
                    'value between 1 and 3", \'8001\')',
```